README.rst

```diff
@@ -0,0 +1,43 @@
+Mozzarella
+==========
+
+This website is a website for clubs and student organizations.
+
+Mozzarella It is written in Python and uses the powerful web framework
+`TurboGears`.
+
+.. TurboGears: http://turbogears.com/
+
+Getting Started
+---------------
+
+These steps are meant to give you an overview of how to get a development
+environment set up for developing this application. Please adapt to your system
+as necessary. `This guide` gives more in-depth instructions on how to start
+working on a TurboGears project.
+
+.. This guide: https://github.com/ColoradoSchoolOfMines/ACM-Guide-list/blob/master/computer_science/TurboGears.md
```

Comment:
Again, this is not valid syntax.

Reply:
Also, why not link to the official TG documentation? If you have improvements to make to the TG documentation, you can submit a PR there (they are good about accepting them)...

Reply:
@jackrosenthal , the point of this is for onboarding people onto the project. That is our number one problem: getting people up to speed quickly. The TG documenation does not do a good job of that. It requires some prior knowledge of what web development is. Our goal with these guides is to help people who have never worked on any web development project get up and running on TurboGears projects. This guide also links directly to the documentation fot TG for after they are set up and understand a little bit of what they are doing.

Reply:
The guide you linked seemed to give a near-exact replica of what is in the README... just remove this entirely then, or link to the TurboGears docs (which gives a pretty good explanation of the virtualenv stuff): http://turbogears.readthedocs.io/en/latest/turbogears/minimal/index.html